### PR TITLE
tar: add Japanese translation

### PR DESCRIPTION
**Pull request by shu-pf**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ja/common/tar.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# tar
+
+> アーカイブ(複数のファイルやフォルダを1つのファイルに纏める)為のユーティリティー。
+> gzipやbzip2などの圧縮方法と組み合わせることが多いです。
+> 詳しくはこちら: <https://www.gnu.org/software/tar>.
+
+-  アーカイブを作成し、それをファイルに書き込む:
```
Comment (navarroaxel) [inline pages.ja/common/tar.md:7]:
```suggestion
- アーカイブを作成し、それをファイルに書き込む:
```

File: pages.ja/common/tar.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`
```
Comment (navarroaxel) [inline pages.ja/common/tar.md:9]: You can translate the file names and `path/to/directory` to Japanese to bring more help non-English speakers.

File: pages.ja/common/tar.md (hunk 1 above)
Comment (shu-pf) [reply pages.ja/common/tar.md]: Thank you for checking.

File: pages.ja/common/tar.md (hunk 2 above)
Comment (shu-pf) [reply pages.ja/common/tar.md]: Thank you for checking.
I fixed it with the following commit.

eb6e21ab3c8e7fb2ba69eed3c1a502ae1859cbd1

File: pages.ja/common/tar.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+> アーカイブ(複数のファイルやフォルダを 1 つのファイルに纏める)為のユーティリティー。
+> gzip や bzip2 などの圧縮方法と組み合わせることが多いです。
+> 詳しくはこちら: <https://www.gnu.org/software/tar>.
+
+- アーカイブを作成し、それにファイルを書き込む:
+
+`tar cf {{出力ファイル名.tar}} {{ファイル1}} {{ファイル2}} {{ファイル3}}`
+
+- gzip 形式で圧縮されたアーカイブを作成し、それにファイルを書き込む:
```
Comment (marchersimon) [inline pages.ja/common/tar.md:11]: Why is `gzip` in there? It isn't in the english version.

Comment (shu-pf) [reply pages.ja/common/tar.md]: I checked it out.

The original file did not say 'gzip', but it existed as a passive system, as in 'g[z]ipped'. I translated it.

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @shu-pf

**Pull request merged**